logistics/scripts/modules/TestHarness.psm1

```diff
@@ -61,6 +61,18 @@ function Get-TestHarnessExecutable {
     return $testHarnessExecutable
 }
 
+function Invoke-SdkGenConsole {
+    Param(
+        [string] $buildConfiguration = "Debug"
+    )
+    
+    $sdkGenConsoleExecutableFolder = "$(Get-RepositoryResolvedPath "\Utilities\SdkGen\EdFi.SdkGen.Console")\bin\$buildConfiguration\netcoreapp3.1"
+    $params = "/c EdFi.SdkGen.Console.exe -m http://localhost:8765/metadata?sdk=true -v 2.4.15 --include-profiles --include-composites --include-identity"
```

Comment:
should this endpoint come from the configuration file?

Reply:
I updated with $apiUrl here